src/components/SearchForm/index.jsx

```diff
@@ -23,6 +23,7 @@ export const SearchForm = () => {
         console.log(e.target[0].value)
         console.log('submitted')
         dispatch({ type: 'UPDATE_NAME', payload: e.target[0].value})
```

Comment:
formData isn't getting used in this file at all, even though you're setting values for it. Seems like you were maybe going for a controlled form, in which you'd set the value in the html input element to formData and on submit send formData in the payload instead of e.target[0].value